lib/restbase.js

```diff
@@ -25,7 +25,7 @@ function RESTBase(options, req) {
         this._parent = par;
         // Remember the request that led to this child instance at each level, so
         // that we can provide nice error reporting and tracing.
-        this._req = par._req;
+        this._req = req;
```

Comment:
@gwicke No, to preserve root request we have a special property, `_rootReq` (line 32). Here the intention is to store a request that was the reason to create this particular child instance. Currently it's used to build out the request chains on recursion depth limit errors to put the chain into the error.